### PR TITLE
[show] Allow system with no ports in config db run without errors

### PR DESCRIPTION
**Pull request by liorghub**
What I did
Allow system with no ports in config db run without errors.
This is needed for modular system which should boot properly without line cards.

How I did it
Do not raise warning if ports dictionary is empty.

How to verify it
Run show interfaces status

Previous command output (if the output of a command-line utility has changed)
root@r-leopard-41:/home/admin# show interfaces status
Warning: failed to retrieve PORT table from ConfigDB!
Warning: failed to retrieve PORT table from ConfigDB!
Interface Lanes Speed MTU FEC Alias Vlan Oper Admin Type Asym PFC
*----------- ------- ------- ----- ----- ------- ------ ------ ------- ------ ----------
root@r-leopard-41:/home/admin#

New command output (if the output of a command-line utility has changed)
root@r-leopard-41:/home/admin# show interfaces status
Interface Lanes Speed MTU FEC Alias Vlan Oper Admin Type Asym PFC
*----------- ------- ------- ----- ----- ------- ------ ------ ------- ------ ----------
root@r-leopard-41:/home/admin#

### CODE REVIEW

**Review by dgsudharsan — DISMISSED** [review]

File: utilities_common/cli.py [hunk 1]
```diff
@@ -127,7 +127,6 @@ def __init__(self, db=None):
 
 
         if not self.port_dict:
-            click.echo(message="Warning: failed to retrieve PORT table from ConfigDB!", err=True)
```
Comment (qiluo-msft) [inline utilities_common/cli.py:130]: >, err=True [](http://example.com/codeflow?start=86&length=10)

This is a good warning message. Can you keep it and just remove `, err=True` ?

Comment (liorghub) [reply utilities_common/cli.py:130]: @qiluo-msft 
I think that if we are going to leave this message, we should at least change it, something like "Configuration database contains no ports" and of course remove err=True.
What do you think?

Comment (qiluo-msft) [reply utilities_common/cli.py:130]: Agree.

Comment (liorghub) [reply utilities_common/cli.py:130]: Done.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**